README.md

```diff
@@ -55,42 +86,45 @@ $ ansible-playbook tests/playbooks/<playbook>
         $ virtctl vnc <vmi_name>
         ```
 
-        > **NOTE:** Use username `kubevirt` and password `kubevirt`.
+        Use the username `kubevirt` and the password `kubevirt`.
 
-### Environment for running playbooks
+5. (Optional) Leave the virtual environment and remove it:
 
-#### Image uploading from localhost (anything using kubevirt_cdi_upload module)
+```shell
+$ deactivate
+$ rm -rf env/
+```
+
+
+### Using the `kubevirt_cdi_upload` module
 
-Your system needs to be able to connect to the [cdi upload proxy pod](https://github.com/kubevirt/containerized-data-importer/blob/master/doc/upload.md). This can be achieved by either:
+To upload an image from localhost by using the `kubevirt_cdi_upload` module, your system needs to be able to connect to the [cdi upload proxy pod](https://github.com/kubevirt/containerized-data-importer/blob/master/doc/upload.md). This can be achieved by either:
 
-1. Exposing the `cdi-uploadproxy` Service from the `cdi` namespace.
+1. Exposing the `cdi-uploadproxy` Service from the `cdi` namespace, or
 
-2. Using `kubectl port-forward` to set up temporary port forwarding through the kubernetes api server, like so: `kubectl port-forward -n cdi service/cdi-uploadproxy 9443:443`
+2. Using `kubectl port-forward` to set up a temporary port forwarding through the Kubernetes API server: `kubectl port-forward -n cdi service/cdi-uploadproxy 9443:443`
 
-### Facts
+### Using the `Facts` module
```

Comment:
Maybe `Using Facts`. We know it is a module

Reply:
you and we know, others don't.